src/components/Board/index.js

```diff
@@ -216,8 +216,10 @@ function BoardContainer({
     if (!coordinates.source) return
 
     isAColumnMove(event.type)
-      ? onColumnDragEnd({ ...coordinates, subject: board.columns[coordinates.source.fromPosition] })
-      : onCardDragEnd({ ...coordinates, subject: getCard(board, coordinates.source) })
+      ? isValidColumnCoordinates(coordinates) &&
+        onColumnDragEnd({ ...coordinates, subject: board.columns[coordinates.source.fromPosition] })
+      : isValidCardCoordinates(coordinates) &&
```

Comment:
The same above (column).

```suggestion
      : isMovingACardToAnotherPosition(coordinates) &&
```